samples/chaincode/simple-asset-go/README.md

```diff
@@ -0,0 +1,257 @@
+	server := &shim.ChaincodeServer{
+		CCID:    ccid,
+		Address: addr,
+		CC:      privateChaincode,
+		TLSProps: shim.TLSProperties{
+			Disabled: true, // just for testing good enough
+		},
+	}
+
+	if err := server.Start(); err != nil {
+		panic(err)
+	}
+}
+```
+
+## Building FPC Go Chaincode 
+
+Create a `Makefile` (i.e., `touch $FPC_PATH/samples/chaincode/simple-asset-go/Makefile`) with the following content: 
+```Makefile
+TOP = ../../..
+include $(TOP)/ecc_go/build.mk
+
+CC_NAME ?= simple-asset
+```
+
+Please make sure that in the file above the variable `TOP` points to the FPC root directory (i.e., `$FPC_PATH`).
+
+In `$FPC_PATH/samples/chaincode/simple-asset-go` directory, to build the chaincode and package it as docker image, execute:
+```bash
+make
+```
+
+After building, you can check your local docker registry that the `fpc/fpc-simple-asset-go` image exists using
+```bash
+docker images | grep simple-asset
+```
+
+
+## Time to test!
+
+Next step is to test the chaincode by invoking transactions, for which you need a basic Fabric network with a channel.
+We will use the test network provided in `$FPC_PATH/samples/deployment/test-network`.
+To invoke the chaincode, we will use the `simple-cli` application in `$FPC_PATH/samples/application/simple-cli-go`
+
+### Enclave Registry
+
+To run FPC chaincode we need to prepare the docker images for the FPC Enclave Registry (ERCC).
+In case you have not yet created them, run `make -C $FPC_PATH/ercc build docker`.
+
+### Prepare the test network
+
+We already provide a detailed tutorial how to use FPC with the test network in [$FPC_PATH/samples/deployment/test-network](../../deployment/test-network).
+However, for completeness let's go through the required steps you need to run once.
+
+```bash
+cd $FPC_PATH/samples/deployment/test-network
+git clone https://github.com/hyperledger/fabric-samples
+cd $FPC_PATH/samples/deployment/test-network/fabric-samples
+curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.3.3 1.4.9 -s
+cd $FPC_PATH/samples/deployment/test-network
+./setup.sh
+```
+
+
+### Start the test network
+
+Now we are ready to launch the fabric test network and install the FPC chaincode on it.
+We begin with setting up the network with a single channel `mychannel`.
+```bash
+cd $FPC_PATH/samples/deployment/test-network/fabric-samples/test-network
+./network.sh up -ca
+./network.sh createChannel -c mychannel
+```
+
+Once the network is up and running, we install the simple asset chaincode and the FPC Enclave Registry.
+We provide a small shell script to make this task a bit easier.
+
+```bash
+export CC_ID=asset-transfer
+export CC_PATH="$FPC_PATH/samples/chaincode/simple-asset-go/"
+export CC_VER=$(cat "$FPC_PATH/samples/chaincode/simple-asset-go/mrenclave")
+cd $FPC_PATH/samples/deployment/test-network
+./installFPC.sh
+```
+
+Note that the `installFPC.sh` script returns an export statement you need now.
+Copy it to your terminal and continue with running `make ercc-ecc-start`.
```

Comment:
make ercc-ecc-start does not execute properly. First I had issues with the name of the test network. I had to change in the /Users/mmunaro/go/src/github.com/hyperledger-labs/fabric-private-chaincode/samples/deployment/test-network/docker-compose.yml file the networks name from fabric_test to net_test which is what my network was called. After correcting this when I execute it I get this error:
```root@docker-desktop:/project/src/github.com/hyperledger/fabric-private-chaincode/samples/deployment/test-network# make ercc-ecc-start
if [ ! -z "/Users/mmunaro/go/src/github.com/hyperledger-labs/fabric-private-chaincode/" ]; then \
  		export SGX_CREDENTIALS_PATH="/Users/mmunaro/go/src/github.com/hyperledger-labs/fabric-private-chaincode//integration/config/ias"; \
  	fi && \
if [ "SIM" = "HW" ]; then \
	export HW_EXTENSION="-hw" \
	    AESMD_PATH="/var/run/aesmd" \
	    SGX_DEVICE_PATH=$(if [ -e "/dev/isgx" ]; then echo "/dev/isgx"; elif [ -e "/dev/sgx/enclave" ]; then echo "/dev/sgx/enclave"; else echo "none"; fi) && \
	[ "${SGX_DEVICE_PATH}" != "none" ] || ( echo "ERROR: SGX_MODE is HW but no sgx device found"; exit 1; ) \
fi && \
env CC_ID=asset-transfer FPC_VERSION=main docker-compose up
Pulling ecc.peer0.org1.example.com (fpc/fpc-asset-transfer:main)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]n
ERROR: pull access denied for fpc/fpc-asset-transfer, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
make: *** [Makefile:29: ercc-ecc-start] Error 1
```
even though before I did as was specified and exported the variable as: `export CC_ID=asset-transfer` I could not find where the fpc as added to the name.